Goals fix

SrutiG opened the pull request:
-Fix member goals frontend
-Add goals default data

Commit:
goals default data

Commit:
fix member goals frontend

daniel-baggerman commented on app/models.py:
```diff
@@ -211,9 +211,9 @@ class Member_Steps(db.Model):
+    username = db.Column(db.String(64), db.ForeignKey('member.username'), primary_key=True)
     proof_verified_by = db.Column(db.String(64))
-    proof_document = db.Column(db.String(64), db.ForeignKey('proof.proof_document'), primary_key=True)
+    proof_document = db.Column(db.String(64))
```
Maybe I believe this proof_doc column no longer needs to be here. Proof_name=proof_doc is my understanding. Is that wrong?

daniel-baggerman replied:
resolved

daniel-baggerman reviewed (approved):
Looks good!

daniel-baggerman committed:
deleted:    app/utils.pyc

daniel-baggerman committed:
modified:   app/app.db
	modified:   app/db_accessor.pyc
	modified:   app/models.pyc
	new file:   app/static/images/profile_pictures/dbagg.jpg
	new file:   app/static/images/profile_pictures/dbaggarstarst.jpg
	new file:   app/static/images/profile_pictures/dbaggearstarst.jpg
	modified:   app/views.pyc

Commit:
fixes
